SparkPost/2web2ui
# AC-1473: Add the Panel Section for SCIM token and Generate SCIM token flow

AC-1473: Add the Panel Section for SCIM token and Generate SCIM token flow

### What Changed
- Added a Panel Section SCIM token under SSO Panel as in the mocks https://sparkpost.invisionapp.com/public/share/FR169HZ78E#/screens/478171293

**When no previous SCIM token is present** 
- Added Generate SCIM Modal which opens on clicking Generate SCIM token.
https://sparkpost.invisionapp.com/public/share/FR169HZ78E#/screens/478171294
- Added "Delete Token" button that appears after adding the token
https://sparkpost.invisionapp.com/public/share/FR169HZ78E#/screens/478171295

**When a SCIM token is already present** - 

- Added the confirmation modal that appears after Clicking on Generate SCIM token, after confirming override, Generate SCIM Modal opens up.
https://sparkpost.invisionapp.com/public/share/FR169HZ78E#/screens/478171296



- Put this section behind a UI feature flag. (sso_scim_section)
- Added actions and reducers for scim token.
- Added cypress tests which help in testing the flow as the apis aren't ready yet.
- SSOPanelSection converted to functional component and unit test updated accordingly.

### How To Test
 - Enable this ui flag - `sso_scim_section`
 - On Account Settings Page if you have SSO Provider set you should be able to see the SCIM Token section as in the mocks
 - Run the cypress tests added in this ticket to verify the flow mentioned above.
 - You can also try clicking Generate New Token on Account settings Page but the Api will throw an error as it is under construction now.

### To Do
